compiler/main/driver.cpp

```diff
@@ -807,6 +904,31 @@ static void readConfig(const ArgumentDescription* desc, const char* arg_unused)
   }
 }
 
+static void setSubInvocation(const ArgumentDescription* desc, const char* arg) {
+  driverInSubInvocation = true;
+}
+
+static void setDriverDebugPhase(const ArgumentDescription* desc,
+                                const char* arg) {
+  driverDebugPhaseSpecified = true;
+
+  if (0 == strcmp(arg, "1")) {
+    driverDebugCompilation = true;
+    driverDebugMakeBinary = false;
+  } else if (0 == strcmp(arg, "2")) {
+    driverDebugCompilation = false;
+    driverDebugMakeBinary = true;
+  } else if (0 == strcmp(arg, "all")) {
+    driverDebugCompilation = true;
+    driverDebugMakeBinary = true;
+  } else {
+    USR_FATAL(
+        "--driver-debug-phase requires a valid phase number or 'all', "
+        "but got: %s\n",
```

Comment:
Perhaps a simpler and more informative message would be:

`"--driver-debug-phase requires either 1, 2, or 'all' as input, but got: %s\n"`

Reply:
I like that phrasing better, will do